velox/common/caching/SsdFile.cpp

```diff
@@ -221,9 +224,10 @@ CoalesceIoStats SsdFile::load(
   for (auto i = 0; i < pins.size(); ++i) {
     auto runSize = ssdPins[i].run().size();
     auto entry = pins[i].checkedEntry();
-    if (runSize > entry->size()) {
-      LOG(INFO) << "IOERR: Requested prefix of SSD cache entry: " << runSize
-                << " entry: " << entry->size();
+    if (FOLLY_UNLIKELY(runSize < entry->size())) {
+      ++stats_.errorsReadSsd;
```

Comment:
Can we add unit tests for all these failure cases? I wonder if we can handle catch these exceptions?